docs/web-apps/automated-testing/testcafe/yaml.md

```diff
@@ -555,9 +555,10 @@ dependencies. If true, package-lock.json will be used during package
 installation which can improve the speed of installation.
 
 To use this feature, additional pre-requisites must be met:
-* A package-lock.json must be present in your project.
-* The `testcafe` version in your package.json must **exactly** match
-the version defined in your saucectl config.
+
```

Comment:
not sure if styling or other things are impacted because of this. Maybe it's worth reverting unnecessary docs changes?